src/Components/Header.css

```diff
@@ -1,9 +1,13 @@
 header.header__menu {
     background: transparent !important;
-    justify-content: space-between;
+    display: block;
     z-index: 1;
 }
 
+header.header__menu > button {
```

Comment:
Just button?

Reply:
yes, we have set that float property on this other file. https://github.com/dotCMS/dotcms-spa-edit-mode/blob/81d2d23096a877697cece13ae532e7a5ca2721ce/src/Components/Nav/Nav.css#L2